agreement/proposalTracker.go

```diff
@@ -144,17 +163,18 @@ func (t *proposalTracker) handle(r routerHandle, p player, e event) event {
 		}
 		t.Duplicate[v.R.Sender] = true
 
+		newFreezer, effect, err := t.Freezer.accept(v)
+		t.Freezer.copyCredentialTrackingState(newFreezer)
 		if t.Staging != bottom {
-			err := errProposalTrackerStaged{}
-			return filteredEvent{T: voteFiltered, Err: makeSerErr(err)}
+			err = errProposalTrackerStaged{}
+			return filteredEvent{T: voteFiltered, CredentialTrackingNote: effect, Err: makeSerErr(err)}
 		}
 
-		var err error
-		t.Freezer, err = t.Freezer.accept(v)
 		if err != nil {
 			err := errProposalTrackerPS{Sub: err}
-			return filteredEvent{T: voteFiltered, Err: makeSerErr(err)}
+			return filteredEvent{T: voteFiltered, CredentialTrackingNote: effect, Err: makeSerErr(err)}
 		}
+		t.Freezer = newFreezer
```

Comment:
Wait, we copied the state from newFreezer into the existing one, then replaced the existing one with the newFreezer? My assumption was that existing freezer becomes eligible for GC after this.